site/source/docs/compiling/WebAssembly.rst

```diff
@@ -69,7 +69,8 @@ upgrade from fastcomp to upstream:
   * Another thing you might notice is that fastcomp's link stage is able to
     perform some types of link time optimization by default that the LLVM
     backend requires flags for (for example,
-    `propagate the "noexcept" property <https://github.com/emscripten-core/emscripten/issues/9817#issuecomment-553459496>`_),
+    `propagate the "noexcept" property <https://github.com/emscripten-core/emscripten/issues/9817#issuecomment-553459496>`_,
+    but that specific regression has been fixed meanwhile),
```

Comment:
If this has been fixed, is there a better example to use here?

Reply:
I can't think of one. Should we remove the section do you think? It is possible though since fastcomp runs GlobalOpt in LLVM during link.

Reply:
I don't think we need to delete the section, but it sounds good to remove this particular example, which may sound confusing. And also it'd be good to mention what the specific flag required for the LLVM is.

Reply:
Thanks, done.